utils/migrate_template_csv_to_rule.py

```diff
@@ -0,0 +1,638 @@
+            groupowner["template"] = f"file_groupowner"
+            groupowner["FILEPATH"] = full_path
+            groupowner["FILEGID"] = gid
+            csv_data[rule_id] = groupowner
+
+        rule_id = f"permissions{path_id}"
+        permissions["template"] = f"permissions"
+        permissions["FILEPATH"] = full_path
+        permissions["FILEMODE"] = mode
+        permissions["FILEUID"] = uid
+        permissions["FILEGID"] = gid
+        csv_data[rule_id] = permissions
+    else:
+        rule_id = f"file_permissions{path_id}"
+        file_permissions["template"] = f"file_regex_permissions"
+        file_permissions["FILENAME"] = file_name
+        file_permissions["FILEPATH"] = dir_path
+        file_permissions["FILEMODE"] = mode
+        csv_data[rule_id] = file_permissions
+
+    # Fields FILEID, STATEMODE, UNIX_DIR, UNIX_FILENAME will be translated into rule.yml
+    # They will be generated from data above during templated content generation
+
+    return permissions
+
+
+def selinux_booleans_csv_to_dict(csv_line, csv_data):
+    selinux_boolean = {}
+
+    sebool_name = csv_line[0]
+    sebool_id = escape_path(sebool_name)
+    rule_id = f"sebool_{sebool_id}"
+
+    sebool_state = csv_line[1]
+    if sebool_state == "on" or sebool_state == "enable":
+        sebool_bool = "true"
+    elif sebool_state == "off" or sebool_state == "disable":
+        sebool_bool = "false"
+
+    selinux_boolean["SEBOOLID"] = sebool_id
+    if sebool_state == "use_var":
+        selinux_boolean["template"] = "sebool_var"
+    else:
+        selinux_boolean["template"] = "sebool"
+        selinux_boolean["SEBOOL_BOOL"] = sebool_bool
+
+    csv_data[rule_id] = selinux_boolean
+    return selinux_boolean
+
+
+def services_disabled_csv_to_dict(csv_line, csv_data):
+    service_disabled = {}
+    service_disabled["template"] = "service_disabled"
+
+    service_name = csv_line[0]
+    package_name = csv_line[1]
+    daemon_name = csv_line[2]
+    mask_service = "true"
+    if len(csv_line) == 4:
+        mask_service = csv_line[3]
+
+    rule_id = f"service_{service_name}_disabled"
+
+    service_disabled["SERVICENAME"] = service_name
+    service_disabled["PACKAGENAME"] = package_name
+    service_disabled["DAEMONNAME"] = daemon_name
+    service_disabled["MASK_SERVICE"] = mask_service
+    csv_data[rule_id] = service_disabled
+    return service_disabled
+
+
+def services_enabled_csv_to_dict(csv_line, csv_data):
+    service_enabled = {}
+    service_enabled["template"] = "service_enabled"
+
+    service_name = csv_line[0]
+    package_name = csv_line[1]
+    if not package_name:
+        package_name = service_name
+    daemon_name = csv_line[2]
+    if not daemon_name:
+        daemon_name = service_name
+
+    rule_id = f"service_{service_name}_enabled"
+
+    service_enabled["SERVICENAME"] = service_name
+    service_enabled["PACKAGENAME"] = package_name
+    service_enabled["DAEMONNAME"] = daemon_name
```

Comment:
Similar to previous comment, do we want content creators to provide all 3 items when they write a new `rule.yml` or do we allow them to provide only service name, meaning the package and daemon will be inferred by the templating engine in that situation?